src/golive/cdn.md

```diff
@@ -52,3 +52,61 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you might not want users to access your platform.sh origin directly. There are 2 ways to secure your origin.
+
+### 1 Basic HTTP Authentication
+
+#### Password protected
+You can password protect your project using [HTTP access control](https://docs.platform.sh/administration/web/configure-environment.html#http-access-control). 
+
+Make sure that you generate a password of sufficient strength. You can then use share the password with your CDN provider. Make sure the CDN adds a header to authenticate correctly to your origin.
+
+
+Simply add a custom header to the origin request with the base64 encoded username:password.
+
+`Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l` 
```

Comment:
This is unclear.  It makes it look like that exact string is what you should add, when it's actually an example. 

Also, "simply" is wording to be avoided.

Reply:
changed.